langing-page/main.css

```diff
@@ -0,0 +1,121 @@
+@import url("https://fonts.googleapis.com/css2?family=Alumni+Sans+Pinstripe&family=Rubik+Marker+Hatch&display=swap");
+.bg {
+  background: url("resources/bg.jpeg") center no-repeat;
+  background-size: cover;
+  background-color: rgba(110, 117, 116, 0.862);
+  background-blend-mode: soft-light;
+  height: 100%;
+  transition-duration: 300ms;
+  color: white;
+}
+
+.dark{
+  background-color: rgba(5, 5, 5, 0.781);
+  transition-duration: 300ms;
+  color: white;
+}
+
+a {
+  font-family: "Alumni Sans Pinstripe";
+  font-weight: bold;
+  font-size: 18px;
+  color: aliceblue;
+  text-decoration: none;
+  padding: 5px 70px 5px;
+  border-radius: 15px;
+  background-color: rgba(54, 45, 172, 0.397);
+  background-blend-mode: light;
+  letter-spacing: 2px;
+  transition-duration: 200ms;
+}
+
+body,
+html {
+  margin: 0;
+  height: 100%;
+}
+
+.description {
+  font-family: "Alumni Sans Pinstripe";
+  font-size: 50px;
+  text-align: center;
+  padding: 150px 150px 200px;
+  height: 80px;
+}
+
+.logoname {
+  font-family: "Alumni Sans Pinstripe";
+  font-weight: bold;
+  font-size: 29px;
+  letter-spacing: 3px;
+  text-align: left;
+  width: 10px;
+  padding: 5px;
+}
+
+.navbar {
+  padding: 10px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  flex-direction: row;
+  -ms-flex: 1; /* IE 10 */
+  flex: 1;
+}
+
+.navbar > div {
```

Comment:
this div has class ```links```. Always use classnames when they are present. In case if there is no classname for particular div, use this method.